Source/Symbolic/Variable.hpp

```diff
@@ -10,15 +10,29 @@
 #include <memory>
 
 #include "Expression.hpp"
+#include "FixedString.hpp"
+
+#define STRINGIZE2(s) #s
+#define STRINGIZE(s) STRINGIZE2(s)
+
+/**
+ * Define a variable with the same name and identifier
+ */
+#define VARIABLE(name) sym::Variable<#name> name
```

Comment:
```suggestion
#define SYM_VARIABLE(name) sym::Variable<#name> name
```
namespaces und macros vertragen sich nicht...